test/README.md

```diff
@@ -0,0 +1,15 @@
+# Running tests
+
+This folder has all the system test files for lxd.
+
+## Dependencies
+
+You'll need the following dependencies installed before you can run them:
+
+    sudo apt-get install jq sqlite3 golang-go
```

Comment:
That list isn't complete, main.sh actually does have the whole list and will return proper errors on missing dependencies, so I'm not sure we want to maintain the list in 4 places (main README.md, main.sh, test/README.md and the packaging).